evm_loader/program/src/instruction/transaction_execute_from_instruction.rs

```diff
@@ -95,21 +99,25 @@ fn execute<'a>(
         let steps_executed = executor.get_steps_executed();
         executor.gasometer_mut().pad_evm_steps(steps_executed);
 
-        let used_gas = executor.used_gas();
-        if used_gas > trx.gas_limit {
-            (evm::ExitError::OutOfGas.into(), vec![], None, trx.gas_limit)
+        let (actions, mut gasometer) = executor.into_state_actions_and_gasometer();
+        let apply = if exit_reason.is_succeed() {
+            Some(actions)
         } else {
-            let apply = if exit_reason.is_succeed() {
-                Some(executor.into_state_actions())
-            } else {
-                None
-            };
+            None
+        };
+
+        let accounts_operations = account_storage.calc_acc_changes(&apply);
+
+        gasometer.record_accounts_operations(&accounts_operations);
 
-            (exit_reason, result, apply, used_gas)
+        let used_gas = gasometer.used_gas();
+        if used_gas > trx.gas_limit {
+            (evm::ExitError::OutOfGas.into(), vec![], None, accounts_operations, trx.gas_limit)

```

Comment:
Same problem here (https://github.com/neonlabsorg/neon-evm/pull/702#discussion_r985120726)

Reply:
Done.

Reply:
Not fixed yet

Reply:
Done.